src/_components/Layouts/ControlBar.tsx

```diff
@@ -184,6 +189,11 @@ export const ControlBar: React.FC<ControlBarProps> = ({
       handleEdit();
     }
   };
+  const onCollaborateClick = () => {
+    if (typeof handleCollaborateMode === 'function') {
+      handleCollaborateMode();
+    }
+  };
```

Comment:
```suggestion
  const onCollaborateClick = () => {
    handleCollaborateMode?.();
  };
```
a little simpler for optional methods. Assuming that what the if was required for anyways